lib/rubocop/ast/node/block_node.rb

```diff
@@ -153,15 +155,16 @@ def void_context?
 
       private
 
-      # Numbered arguments of this `numblock`.
       def numbered_arguments
-        return [].freeze unless numblock_type?
-
         max_param = children[1]
         1.upto(max_param).map do |i|
           ArgNode.new(:arg, [:"_#{i}"])
         end.freeze
       end
+
+      def it_arguments
+        [ArgNode.new(:arg, [:it])].freeze
```

Comment:
It may be possible to consider making it a constant.

Reply:
That works, sure